_content/essays/karma_geoff.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Karma"
+authors:
+  - geoff
+external_url: "https://www.accesstoinsight.org/lib/authors/thanissaro/karma.html"
```

Comment:
Yeah, AtI `source` and dhammatalks.org `external` URLs are preferred for Than Geoff's essays 😁 Thanks